src/js/interface.js

```diff
@@ -456,6 +459,16 @@ window.addEventListener("load", function() {
                 if (value===10) {
                   return document.createElement("br");
                 } else {
+                  if(value>=-10240 && value<=-9217) {
```

Comment:
Might be handy to add a comment explaining where the values come from.

Reply:
I can do that. It'll be lengthy, though, as it involves the stupidity (sorry, idiosyncrasy) of JavaScript only having a single (floating point) number type, 16-bit twos complement, and the way UTF-16 specifies the encoding of non-BMP characters via surrogate pairs.